recipes/portcullis/1.1.2/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "portcullis" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "5c581a7f827ffeecfe68107b7fe27ed60108325fd2f86a79d93f61b328687749" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/EI-CoreBioinformatics/portcullis/archive/Release-1.1.2.tar.gz
+  sha256: {{ sha256 }}
+
+about:
+  home: https://github.com/maplesond/portcullis
+  license: GPL3
+  license_file: COPYING
+  summary: 'Splice junction analysis and filtering from BAM files'
+
+build:
+  number: 5
+  skip: True # [osx or py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - libtool
+    - boost
+    - zlib
+    - samtools >=1.9
+    - perl
+    - setuptools
+    - tabulate
+    - numpy >=1.15
+    - pandas
+    - samtools >=1.9
+    - python    
+  host:
+    - autoconf
```

Comment:
I’m myself not 100% sure, yet, about what should go into `build:` and `run:` requirements, but [according to the documentation](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#requirements-section), build tools such as `autoconf`, `automake` and compilers should be listed under `build:`. Anything specific to the target system such as libraries needs to be under `host:`.